indicators/NO_SLIT_001-004/R/NO_SLIT_001-004.qmd

```diff
@@ -0,0 +1,959 @@
+#| echo: false
+source(here::here("_common.R"))
+```
+
+```{r}
+#| echo: false
+meta <- readxl::read_xlsx("../metadata.xlsx")
+st <- meta |>
+  filter(Variable == "status") |>
+  pull(Value)
+version <- meta |>
+  filter(Variable == "Version") |>
+  pull(Value)
+auth <- meta |>
+  filter(Variable == "authors") |>
+  pull(Value)
+year <- meta |>
+  filter(Variable == "yearAdded") |>
+  pull(Value)
+id <- meta |>
+  filter(Variable == "indicatorID") |>
+  pull(Value)
+name <- meta |>
+  filter(Variable == "indicatorName") |>
+  pull(Value)
+url <- meta |>
+  filter(Variable == "url") |>
+  pull(Value)
+
+meta <- meta |>
+  mutate(Variable = case_match(Variable,
+    "indicatorID" ~ "Indicator ID" ,
+    "indicatorName" ~ "Indicator Name",
+    "country" ~ "Country",
+    "continent" ~ "Continent",
+    "ECT" ~ "Ecosystem Condition Typology Class",
+    "yearAdded" ~ "Year added",
+    "yearLastUpdate" ~ "Last update",
+    .default = Variable
+   )
+  ) |>
+  filter(Variable != "authors")
+
+```
+
+<!--# The following parts are autogenerated. Do not edit. -->
+
+```{r}
+#| echo: false
+#| results: asis
+status(st)
+```
+
+::: {layout-ncol="2"}
+
+
+
+> **Recomended citation**: `r paste(auth, " ", year, ". ", name, " (ID: ", id, ") ", "v. ", version, ". ecRxiv: https://view.nina.no/ecRxiv/", sep="")`
+
+> **Version**: `r version`
+
+:::
+
+```{=html}
+<details>
+<summary>Show metadata</summary>
+```
+
+```{r tbl-meta}
+#| tbl-cap: 'Indicator metadata'
+#| echo: false
+#| warning: false
+
+meta |>
+  select(Variable, Value) |>
+  kbl(col.names = NULL) 
+
+```
+
+```{=html}
+</details>
+```
+
+::: {.callout-tip collapse="true"}
+
+## Logg
+
+<!--# Update this logg with short messages for each update -->
+- 09 Apr. 2025 - Original PR
+:::
+
+
+<hr />
+
+<!--# Document you work below.  -->
+
+## 1. Summary
+
+The 'Erosive disturbance' indicator describes to what degree an ecosystem is affected by human-caused disturbances, mainly linked to erosion from heavy vehicles or damage from intensive human use, based on the 'tungekjoretoy' and 'slitasje' variables registered in ANO monitoring sites. The indicator is one-sided as a lack of this kind of erosive disturbance obviously is not a problem. The reference condition is defined as an ecosystem without any erosive disturbance (applies to all types of ecosystems and all regions), and thus the reference threshold for scaling is set to 0 disturbance at a site. The reference level X~60~, i.e. the threshold between good and reduced ecological condition, as well as X~0~, are set on an expert-judgement basis: less than 1/16th of the area affected by both heavy vehicles and overuse represents good condition, and more than 1/2 of the area for one of heavy vehicles or overuse plus up to 1/2 of the area for the other represents X~0~ where the ecosystem for sure has transformed into something else (see also Nybø et al. 2023 for an implementation of this indicator for Norwegian wetland and open lowland ecosystems).
```

Comment:
Good summary. 
The documentation has been written using an old template, but I will not try and convert to the new template now.